Code/Java/Bucket_Sort.java

```diff
@@ -0,0 +1,133 @@
+/*
+Bucket Sort is a sorting technique that sorts the elements by first dividing the elements into several groups called buckets.
+The elements inside each bucket are sorted using any of the suitable sorting algorithms or recursively calling the same algorithm.
+Several buckets are created. Each bucket is filled with a specific range of elements. 
+The elements inside the bucket are sorted using any other algorithm. Finally, the elements of the bucket are gathered to get the sorted array.
+
+*/
+
+
+import java.util.*;
+
+class BucketSort {
+
+    private static void bucketSort(float[] arr, int k) {
+        if (arr.length < 2)
+            return;
+        
+        // getting upper limit for dividing numbers in buckets
+        int length = arr.length;
+        float max_val = arr[0];
+        for (int i = 1; i<length; i++) {
```

Comment:
Add space around the operators like <
```suggestion
        for (int i = 1; i < length; i++) {
```